src/encoded/static/components/cart/cart.js

```diff
@@ -1581,6 +1583,19 @@ const sortPseudoDefaultFilesByStatus = (files) => (
 );
 
 
+/**
+ * Sort an array of assemblies, newer assemblies first. Carts don’t track genome_annotation, so those
+ * don't enter into this sorting.
+ * @param {array} files Files to sort
```

Comment:
the parameter is "assemblies" not "files"

Reply:
👍